source/direct_debit/index.html.md.erb

```diff
@@ -255,96 +133,197 @@ Required arguments:
+payment each time a subsequent one is requested. It then takes a further 1
+working day before the payment reaches the service’s bank account.
 
-GOV.UK Pay will send an email to the user giving them 3 working days' advance
+GOV.UK Pay will send an email to the user giving them 2 working days' advance
 notification that payment will be collected against their mandate. The email will contain the mandate reference number.
 
 You can collect at any time interval. You may collect between £1 and £5000 each
-time.
+time. [Contact us](https://docs.payments.service.gov.uk/support_contact_and_more_information/#contact-us) if you need to collect more than £5000.
 
-## Direct Debit payment statuses
+### Get information about a payment
 
-These apply to both one-off and variable Direct Debit payments.
+The call to check the status of a payment with the GOV.UK Pay API is:
+
+`GET /v1/directdebit/payments/<paymentid>`
+
+If the payment exists, the API response will include a `state` object. This includes the following parameters:
+
+- [`status`](#payment-statuses)
+- [`details`](https://developer.gocardless.com/api-reference/#events-payment-causes), which is a reason for the status from GoCardless
+- `finished`, which will be `true` if GOV.UK Pay has successfully collected the payment
+
+Refer to the <a
+href="https://govukpay-api-browser.cloudapps.digital/#newpayment"
+target="blank">GOV.UK Pay API browser</a> or the <a href="https://github.com/alphagov/pay-publicapi/blob/master/swagger/swagger.json" target="blank">Swagger file</a> for more
+information on the API response
+
+You can also check the status of a payment by signing into your GoCardless account through the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login).
+
+## Generate a list of mandates or payments
+
+You can search mandates or payments by using the API.
+
+### Search mandates
+
+The call to search mandates with the GOV.UK Pay API is:
+
+`GET /v1/directdebit/mandates?<queryparameters>`
+
+For example:
+
+`GET /v1/directdebit/mandates?from_date=2018-10-01T13:30:00Z&to_date=2018-10-14T15:00:00Z&reference=12345&status=pending`
+
+Some of the `queryparameters` you can use to search are:
+
+- `reference`
+- `status`
+- `bank_statement_reference`
+- `email`
+- `name`
+
+All the parameters are case insensitive. You can search for partial values of `reference`, `email` and `name`.
+
+If you search for a specific mandate, all criteria you use must match. Otherwise that mandate will not be returned in the results.
+
+If your request has no query parameters, the API will return all mandates.
+
+### Search payments
+
+The call to search payments with the GOV.UK Pay API is:
+
+`GET /v1/directdebit/payments?<queryparameters>`
+
+For example:
+
+`GET /v1/directdebit/payments?from_date=2018-10-01T13:30:00Z&to_date=2018-10-14T15:00:00Z&reference=12345&status=success`
+
+Some of the `queryparameters` you can use to search are:
+
+- `reference`
```

Comment:
I think `reference` is a substring search for payments too.

Reply:
Thank you - added in https://github.com/alphagov/pay-tech-docs/pull/298/commits/562c05d6cc099a67190a4bfe298a6acc6291794c.